src/components/home/Home.jsx

```diff
@@ -45,6 +46,16 @@ const Home = ({ route }) => {
       console.error('Error retrieving weather data frontend:', error);
     }
   };
+  const getPlaylist = async(userData) => {
+    try{
+      const res = await axios.get(`${constants.BACKEND_DEV_SERVER_URL}/playlist?token=${token}&weather_cond=${apiResponse.currentConditions.icon}`);
+      setUserPlaylist(res.data);
```

Comment:
why send weather conditions parameter in this API call?

Reply:
using the keyword retrieved from weather API, and passing it to the backend. In the backend, we use the Spotify search API to retrieve the first playlist with a description containing the weather keyword, like "cloudy"